PayPalMessaging/src/main/java/com/braintreepayments/api/PayPalMessagingListener.kt

```diff
@@ -0,0 +1,34 @@
+package com.braintreepayments.api
+
+/**
+ * Callback used to communicate [PayPalMessagingView] events
+ * Note: **This module is in beta. It's public API may change or be removed in future releases.**
+ */
+interface PayPalMessagingListener {
+
+    /**
+     * Called when the user has clicked the [PayPalMessagingView].
+     */
+    fun onClick()
```

Comment:
```suggestion
    fun onPayPalMessagingClick()
```
nit: I would add module prefixes here to avoid potential collisions with other SDK interface methods.

We've done this in the past to support merchant apps that use a single class to implement multiple SDK interfaces e.g. `MyFragment implements PayPalMessagingListener, PayPalWebListener...`.

Reply:
That makes total sense, updated: https://github.com/braintree/braintree_android/pull/895/commits/0f95127622ceb09d656933c5c373d8a93eb4770e